supporting_scripts/course-scripts/quick-course-setup/large_course_main.py

```diff
@@ -0,0 +1,72 @@
+import requests
+import configparser
+from logging_config import logging
+from requests import Session
+from utils import authenticate_user
+from create_course import create_course
+from create_users import create_students, user_credentials
+from add_users_to_course import add_students_to_groups_of_course
+from manage_programming_exercise import create_programming_exercise, add_participation, commit, exercise_ids
+from delete_students import delete_students
+
+# Load configuration and constants
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+# Constants
+STUDENTS_TO_CREATE: int = int(config.get('Settings', 'students')) + 1
+COMMITS_PER_STUDENT: int = int(config.get('Settings', 'commits'))
+EXERCISES_TO_CREATE: int = int(config.get('Settings', 'exercises'))
+
+CLIENT_URL: str = config.get('Settings', 'client_url')
+SERVER_URL: str = config.get('Settings', 'server_url')
+ADMIN_USER: str = config.get('Settings', 'admin_user')
+ADMIN_PASSWORD: str = config.get('Settings', 'admin_password')
+COURSE_NAME: str = config.get('CourseSettings', 'course_name')
+COURSE_ID: str = config.get('CourseSettings', 'course_id')
+IS_LOCAL_COURSE: bool = config.get('CourseSettings', 'is_local_course').lower() == 'true'
+
+def delete_all_created_students(session: Session) -> None:
+    authenticate_user(ADMIN_USER, ADMIN_PASSWORD, session)
+    delete_students(session, CLIENT_URL)
+    logging.info(f"Deleted all created students successfully")
+
+def main() -> None:
+    # Step 1: Authenticate as admin
+    session: Session = requests.session()
+    authenticate_user(ADMIN_USER, ADMIN_PASSWORD, session)
+
+    # Step 2: Create users
+    create_students(session, STUDENTS_TO_CREATE)
+
+    # Step 3: Create a course
+    created_course_response = create_course(session)
```

Comment:
When running the script locally and using it twice, I got the error

```
2024-09-04 07:07:13,719 - ERROR - Problem with the group 'students' and interacting with a test server? Is 'is_local_course' in 'config.ini' set to 'False'?
Exception: Could not create course Local Course; Status code: 400
Double check whether the courseShortName LocalCourse is valid (e.g. no special characters such as '-') not already used for another course!
```

The workaround is to manually delete the course. Is there any way to remove the course apart from that?